css-borders-4/Overview.bs

```diff
@@ -1459,101 +1459,154 @@ Rendering 'corner-shape'</h4>
 When rendering elements with shaped corners, the element's path needs to be offset,
```

Comment:
We should use the newly established term "border contour path" throughout the spec. Otherwise it might be unclear what path is actually meant.

```suggestion
When rendering elements with shaped corners, the element's [=border contour path=] needs to be offset,
```